### PR TITLE
Adding Brazilian Portuguese books

### PR DESCRIPTION
**Pull request by heltonricardo**
## What does this PR do?
Add resource(s)

### Description

Brazilian Portuguese Books:
- Linguagem Assembly: Introdução ao padrão Intel 8086 - José Augusto N. G. Manzano (@J-AugustoManzano)
- Programação de Computadores com Linguagem Julia - José Augusto N. G. Manzano (@J-AugustoManzano)
- Linguagem LISP - Primeiros passos com Common LISP (CL) - José Augusto N. G. Manzano (@J-AugustoManzano)
- Academia da Tartaruga - José Augusto N. G. Manzano (@J-AugustoManzano)
- Linguagem Logo: Introdução com UCBLogo 6.2.2 - José Augusto N. G. Manzano (@J-AugustoManzano)
- Programação de computadores para iniciantes com Small Basic - José Augusto N. G. Manzano (@J-AugustoManzano)

### Why is this valuable (or not)?

Books with high quality content used even in universities

### How do we know it's really free?

The author has published them on his GitHub and he is aware of publishing them in this repository. I used to be his student.

### For book lists, is it a book? For course lists, is it a course? etc.

All are books.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by J-AugustoManzano** [comment]
To Whom It May Concern: I am aware of the indication and free distribution of these books of my authorship. Authorized distribution.

**Review by eshellman — CHANGES_REQUESTED** [review]
Nice additions. Other than using the more direct url for the Lisp book, consider whether students are going to find the Small Basic book under "Small Basic" if they are looking for a Basic book. Remember our headings are to help people find what they want - they are not for experts who already understand the difference between dialects.

File: books/free-programming-books-pt_BR.md [hunk 1]
```diff
@@ -99,6 +103,14 @@
 
 * [Assembly x86](https://mentebinaria.gitbook.io/assembly-x86/) - Luis Felipe, Mente Binária (gitbook)
 * [Execução de Programas: Como funcionam carregadores, ligadores e interpretadores](https://www.inf.ufpr.br/bmuller/assets/docs/livroSB.pdf) - Bruno Müller Junior (PDF)
+* [Linguagem Assembly: Introdução ao padrão Intel 8086](https://github.com/J-AugustoManzano/livro_Assembly-Intro-8086) - José Augusto N. G. Manzano (PDF)
+
+
+### Basic
+
+#### Small Basic
```
Comment (eshellman) [inline books/free-programming-books-pt_BR.md:111]: just one heading please "Basic" is close enough

File: books/free-programming-books-pt_BR.md [hunk 2]
```diff
@@ -8,6 +8,8 @@
 * [Android](#android)
 * [Arduino](#arduino)
 * [Assembly](#assembly)
+* [Basic](#basic)
+    * [Small Basic](#small-basic)
```
Comment (eshellman) [inline books/free-programming-books-pt_BR.md:12]: just one heading please "Basic" is close enough

**Review by eshellman — CHANGES_REQUESTED** [review]
If we had a bunch of Basic books, having a subcategory for Small Basic would be ok. But the title is "Small Basic, so having the subcategory is overkill. Imagine if you only have "WATFIV Fortran" and no other FORTRAN books. You would just expect a singular category for FORTRAN without a WATFIV subcategory.

**Comment by heltonricardo** [comment]
Thanks for the feedback. Correction has been made and pull request is updated.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**